runtime/doc/editing.txt

```diff
@@ -1236,29 +1244,50 @@ present in 'cpoptions' and "!" is not used in the command.
 							*:chd* *:chdir*
 :chd[ir][!] [path]	Same as |:cd|.
 
+							*:tc* *:tcd*
+:tc[d][!] {path}        Like |:cd|, but set the current directory for the
+                        current tab.  The current directory for other tabs
+                        or the local directory of windows in this tab is not
+                        changed, except for the current window.
+
+							*:tcd-*
+:tcd[!] -		Change to the previous current directory (before the
+			previous ":tcd {path}" command).
+
+							*:tch* *:tchdir*
+:tch[dir][!]		Same as |:tcd|.
+
 							*:lc* *:lcd*
 :lc[d][!] {path}	Like |:cd|, but only set the current directory for the
 			current window.  The current directory for other
-			windows is not changed.
+			windows or any tabs is not changed.
```

Comment:
@kopischke OK, I'm not surprised, I just didn't know. I never used VimScript outside of the usual `.vimrc` settings, and it doesn't help that the language has a huge standard library and more edge cases than a stairway. Is there book to recommend? I read *The VimL Primer* by Benjamin Klein, but it's really just an introduction, something that should be the first two chapters of a much larger book.

@justinmk And what if yet another scope gets added in the future? Off the top of my head I could imagine a UI managing several instances of NeoVim and we would need an `:icd` command. Then the unified `haslocaldir()` would need to return a three-bit number:
~~~~
0b000 "global
0b001 "window
0b010 "tab
0b011 "window & tab
0b100 "instance
0b101 "instance & window
0b110 "instance & tab
0b111 "instance & tab & window
~~~~
So plugin authors would have to update their plugins *again*. Whereas if they `AND` the result with `1` it will work for all eternity.